README.md

```diff
@@ -28,7 +28,7 @@ if Rails.env.development?
     config.links = [
       { name: "Routes", path: "/rails/info/routes" },
       { name: "Database", path: "/rails/db" }, # rails_db gem must be installed
-      { name: "Data Model", path: "/erd.png" }, # erd.png must be in public/ folder
+      { name: "ERD", path: "/erd" }, # erd.png must be in the root folder
```

Comment:
I forgot to edit the `dev_toolbar` initializer in my test repo, which confused me for a bit!

Reply:
Yeah, that's why I'm going to merge this proj syncing update: https://github.com/firstdraft/project-syncing/pull/19, immediately following this update here.